phpstan.neon

```diff
@@ -31,6 +31,11 @@ parameters:
             # Annotations in Codeception are not well describing
             message: '#^Parameter \#(\d+) \$cssOrXPath of method Tests\\(.*)\\Codeception\\(.*) expects (.*)#'
             path: %currentWorkingDirectory%/packages/framework/tests/Test/Codeception/*
+        -
+            # the reported code is not used anywhere but can not be changed (the class is deprecated and will be removed in next major)
```

Comment:
_nitpick_ what about use `/** @phpstan-ignore-next-line */` in the `AbstractNativeQuery`?

that way it's not necessary to have it in two files (monorepo + framework) and the ignore will perish along with the class.

Reply:
that is a great idea :ok_hand: